container/webui/run_openqa.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -e
+set -x
```

Comment:
I guess you added that for temporary debugging

Reply:
yes, I'll remove this